### PR TITLE
mumble: 1.4.287 -> 1.5.517

### PR DESCRIPTION
**Pull request by ajs124**
###### Description of changes
Contains  #199111

We can wait for a stable (non RC) release of mumble, if the maintainers prefer that.

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -23169,18 +23169,6 @@ with pkgs;
     inherit lib pkgs qt5;
   });
 
-  # TODO: remove once no package needs this anymore or together with OpenSSL 1.1
-  # Current users: mumble, murmur
-  qt5_openssl_1_1 = qt5.overrideScope (_: super: {
```
Comment (infinisil) [inline pkgs/top-level/all-packages.nix:23174]: This should at least be replaced with a `throw` in `aliases.nix`

**Review by infinisil — CHANGES_REQUESTED** [review]

**Comment by felixsinger1** [comment]
I don't want to start that discussion here, but not sure how to reach out else.

I am @felixsinger writing from this account because I am unable to review this PR and write comments with my other account, while I am one of the maintainers for this package. I am able to review other PRs and others are able to review this PR as well. However, if you (@ajs124) blocked me somehow, please remove that. If you got a problem with me then please reach out to me and let's try resolving it, but blocking me and making me unable to review PRs doesn't seem very reasonable.

**Comment by felixsinger1** [comment]
I talked to the Mumble developers. This release candidate shouldn't be used since it has issues which need to be fixed. Due to lack of time it takes longer than expected.

**Review by ghost — CHANGES_REQUESTED** [review]
I talked to the Mumble developers. This release candidate shouldn't be used since it has issues which need to be fixed. Due to lack of time it takes longer than expected. So I'm against merging this.

**Pull request closed without merging**